week1/project/ecomerce/src/App.js

```diff
@@ -0,0 +1,18 @@
+import './App.css';
+
+import {useState} from 'react';
+import Category from './components/Category';
+import Products from './components/Products';
+
+function App() {
+  const [category, setCategory] = useState('All');
+  return (
+    <div className="App">
+      <h1>Products</h1>
+      <Category useStateCategory={setCategory} />
```

Comment:
naming is very important .. `useStateCategory` is for custom hooks which you will study later how to create a hook yourself so using `use` keyword will confuse you or your team when you read it.

✏️need to modify✏️

Reply:
changed